syntax				= "proto3";
option optimize_for = CODE_SIZE;


message Cfg_Phase
{
	uint32 idmap	 = 1; // 地图ID
	uint32 idphase   = 2; //位面ID
	float  viewgrid_width = 3; //视野格子默认宽度
	float  left     = 5; //左上角x
	float  top      = 6; //左上角y
	float  right    = 7; //右下角x
	float  bottom   = 8; //右下角x
	uint32 link_phase = 9; //与哪个位面直接相连（共享SceneTree）
}
